src/main/resources/modules/wearables_ecg.json

```diff
@@ -0,0 +1,603 @@
+        "low": 20,
+        "high": 29,
+        "unit": "years"
+      },
+      "name": "Age_20_29"
+    },
+    "Age_30_39": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 30,
+        "high": 39,
+        "unit": "years"
+      },
+      "name": "Age_30_39"
+    },
+    "Age_40_49": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 40,
+        "high": 49,
+        "unit": "years"
+      },
+      "name": "Age_40_49"
+    },
+    "Age_50_59": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 50,
+        "high": 59,
+        "unit": "years"
+      },
+      "name": "Age_50_59"
+    },
+    "Age_60_69": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 60,
+        "high": 69,
+        "unit": "years"
+      },
+      "name": "Age_60_69"
+    },
+    "Age_70_79": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 70,
+        "high": 79,
+        "unit": "years"
+      },
+      "name": "Age_70_79"
+    },
+    "Age_80_99": {
+      "type": "Delay",
+      "direct_transition": "Sedentary_Job",
+      "range": {
+        "low": 80,
+        "high": 99,
+        "unit": "years"
+      },
+      "name": "Age_80_99"
+    },
+    "Sedentary_Job": {
+      "type": "ConditionOnset",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 160923008,
+          "display": "Sedentary job (finding)",
+          "value_set": ""
+        }
+      ],
+      "distributed_transition": [
+        {
+          "transition": "Sedentary_Job_End",
+          "distribution": 0.75
+        },
+        {
+          "transition": "Sedentary_Lifestyle",
+          "distribution": 0.25
+        }
+      ],
+      "name": "Sedentary_Job"
+    },
+    "Sedentary_Lifestyle": {
+      "type": "ConditionOnset",
+      "target_encounter": "Wellness_Baseline_Encounter",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 415510005,
+          "display": "Sedentary lifestyle (finding)",
+          "value_set": ""
+        }
+      ],
+      "remarks": [
+        "More than 60% of US adults do not engage in recommended amount of physical activity.  25% of adults are not physically active at all.",
+        "https://www.cdc.gov/nccdphp/sgr/adults.htm#:~:text=More%20than%2060%20percent%20of,Women%20than%20men."
+      ],
+      "distributed_transition": [
+        {
+          "transition": "Wellness_Baseline_Encounter",
+          "distribution": 0.2
+        },
+        {
+          "transition": "Symptom_Onset_Delay",
+          "distribution": 0.79
+        },
+        {
+          "transition": "Sedentary_Lifestyle_End",
+          "distribution": 0.01
+        }
+      ],
+      "name": "Sedentary_Lifestyle"
+    },
+    "Symptom_Onset_Delay": {
```

Comment:
The symptoms that get added in this sequence are immediately ended right after, so they don't really do anything. Then the ConditionEnds after that aren't going to work because they point to a Guard instead of a ConditionOnset. I'd suggest just having this whole branch (the "80%" after the guard) go directly to Terminal